cms/tests/toolbar.py

```diff
@@ -1467,7 +1467,7 @@ def test_edit_page(self):
                 page.pk, page.get_title(language)))
         self.assertContains(
             response,
-            '<div class="cms_plugin cms_plugin-cms-page-changelist-%s cms_render_model cms_render_model_block"><h3>Menu</h3></div>' % page.pk)
+            '<div class="cms_plugin cms_plugin-cms-page-changelist-%s cms_render_model cms_render_model_block">\n        <h3>Menu</h3>' % page.pk)
         self.assertContains(
```

Comment:
the change was deliberate not to clutter the  assert string with `\n    \n  {not sure about amount of space symbols}      \n</div>`  
it would probably make sense to not compare html as strings, but rather as a tree of dom nodes in future